### ISSUE TITLE
No maping file found issue

### ISSUE THREAD

**Issue by Shibon-ll** [akeneo/pim-community-dev#11538]
Akeneo  system show's error in console  while update the product 

> No mapping file found named 'VariantProduct.orm.yml' for class 'Pim\Component\Catalog\Model\VariantProduct'.

After clearing the cache it will works fine 

![image](https://user-images.githubusercontent.com/45478471/74003846-066fc180-499a-11ea-9f11-68ddd225e65f.png)

Akeneo version 2.3

**Comment by Doodoune** [comment]
Hi @Shibon-ll,

Do you have this error after a migration? Or directly after installing a new PIM in 2.3 version?

Regards,

**Comment by Shibon-ll** [comment]
@Doodoune  After installing i am trying to access all product related to category

**Comment by Doodoune** [comment]
Did you follow the installation process?https://docs.akeneo.com/3.2/install_pim/manual/installation_ce_archive.html

In this process, one of the steps is to clean your cache.

Regards,

**Comment by Shibon-ll** [comment]
Yes i have cleared cache but after some times this error will occurs

**Comment by Doodoune** [comment]
Did this error occur regularly? I won't reproduce it on our reference environment.

**Comment by Shibon-ll** [comment]
No it will occurs randomly.. After clearing cache it will works... but need to clear cache for each 2 days
